postprocessing/pp-opus.c

```diff
@@ -85,6 +85,15 @@ int janus_pp_opus_process(FILE *file, janus_pp_frame_packet *list, int *working)
 			int i=0;
 			for(i=0; i<silence_count; i++) {
 				pos = (tmp->prev->ts - list->ts) / 48 / 20 + i + 1;
+				if(tmp->next == NULL) {
+					JANUS_LOG(LOG_WARN, "[SKIP] pos: %06" SCNu64 ", skipping remaining silence, next is NULL\n", pos);
```

Comment:
I'm not sure this is ok. You're dropping the last audio packet, this way.

Reply:
Adjusted in next commit